src/collector/top_score_collector.rs

```diff
@@ -718,7 +718,7 @@ impl SegmentCollector for TopScoreSegmentCollector {
         self.0.collect(doc, score);
     }
 
-    fn harvest(self) -> Vec<(Score, DocAddress)> {
```

Comment:
this is not helpful

Reply:
Whoops sorry about that. Will revert in a later commit